partitura/io/mei_export_v2.py

```diff
@@ -0,0 +1,383 @@
+            else:
+                score_def_el.set('sig', str(abs(key_sig.fifths)) + 'f')
+            # Find the pname from the number of sharps or flats and the mode
+            score_def_el.set('pname', fifths_mode_to_key_name(key_sig.fifths, key_sig.mode).lower())
+            # Add the scoreDef element at before the measure element starts
+            parent = measure_el.getparent()
+            parent.insert(parent.index(measure_el), score_def_el)
+
+    def _handle_ts_changes(self, measure_el, start, end):
+        # For key signature changes, we add a new scoreDef element at the beginning of the measure
+        # and add the key signature element as attributes of the scoreDef element
+        for time_sig in self.part.iter_all(spt.TimeSignature, start=start, end=end):
+            if time_sig.start.t == 0:
+                continue
+            # Create the scoreDef element
+            score_def_el = etree.Element('scoreDef')
+            score_def_el.set(XMLNS_ID, "scoredef-" + self.elc_id())
+
+            # Add the scoreDef element at before the measure element starts
+            parent = measure_el.getparent()
+            parent.insert(parent.index(measure_el), score_def_el)
+            score_def_el.set('count', str(time_sig.beats))
+            score_def_el.set('unit', str(time_sig.beat_type))
+
+    def _handle_harmony(self, measure_el, start, end):
+        # For key signature changes, we add a new scoreDef element at the beginning of the measure
+        # and add the key signature element as attributes of the scoreDef element
+        for harmony in self.part.iter_all(spt.RomanNumeral, start=start, end=end):
+            harm_el = etree.SubElement(measure_el, 'harm')
+            harm_el.set(XMLNS_ID, "harm-" + self.elc_id())
+            harm_el.set("staff", str(self.part.number_of_staves))
+            harm_el.set("tstamp", str(np.diff(self.part.quarter_map([start, harmony.start.t]))[0]+1))
+            harm_el.set("place", "below")
+            # text is a child element of harmony but not a xml element
+            harm_el.text = harmony.text
+
+
+@deprecated_alias(parts="score_data")
+def save_mei(
+    score_data: spt.ScoreLike,
+    out: Optional[PathLike] = None,
+) -> Optional[str]:
+    """
+    Save a one or more Part or PartGroup instances in MEI format.
+
+    Parameters
+    ----------
+    score_data : Score, list, Part, or PartGroup
+        The musical score to be saved. A :class:`partitura.score.Score` object,
+        a :class:`partitura.score.Part`, a :class:`partitura.score.PartGroup` or
+        a list of these.
+    out: str, file-like object, or None, optional
+        Output file
+
+    Returns
+    -------
+    None or str
+        If no output file is specified using `out` the function returns the
+        MEI data as a string. Otherwise the function returns None.
+    """
+
+    if isinstance(score_data, spt.Score):
+        score_data = spt.merge_parts(score_data.parts)
```

Comment:
I think it will not work here, because merge_part with default parameters will put all parts in a single staff. I would change this check and if the input contains more than one part, you raise an exception, that this is yet not supported. Something like
```
if isinstance(parts, Score):
        parts = parts.parts
    else:
        parts = list(iter_parts(parts)
if len(parts) > 1:
    raise ....
```

Reply:
Addressed with commit [957df35](https://github.com/CPJKU/partitura/pull/347/commits/957df357accb8f1246e273afb8dfd2de592db4ef)